src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -109,14 +110,17 @@ struct OperationParameters {
 struct Params {
     // Query Params/Header in initializer
     var required: [KeyValueViewModel]
-    // Query Params/Header need to add Nil check
+    // Query Params/Header which is pull from Options struct need to add Nil check
     var optional: [KeyValueViewModel]
+    // Query Params/Header which is pull from Optional Client property. It need to add Nil check
+    var optionalClientProperty: [KeyValueViewModel]
```

Comment:
Not sure I agree with this approach. It seems the value side of the KeyValueViewModel should know whether the value is pulled from the client (`client.value`), the signature (`value`) or a constant (`"value"`)

Reply:
Reverted the change.